com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/HDRenderPipeline.VolumetricCloudsSky.cs

```diff
@@ -274,7 +276,7 @@ internal void RenderVolumetricClouds_Sky(CommandBuffer cmd, HDCamera hdCamera, M
                         parameters.commonData.cloudsCB._CloudsPixelCoordToViewDirWS = pixelCoordToViewDir[faceIdx];
 
                         // Render the face straight to the output cubemap
-                        RenderVolumetricClouds_Sky_High(cmd, parameters, m_MpbClouds, m_IntermediateCloudsLighting0Buffer, m_IntermediateCloudsLighting1Buffer, m_IntermediateCloudsDepthBuffer, skyboxCubemap);
+                        RenderVolumetricClouds_Sky_High(cmd, parameters, m_MpbClouds, m_IntermediateCloudsLighting0Buffer, m_IntermediateCloudsLighting1Buffer, m_IntermediateCloudsDepthBuffer, skyboxCubemap, TextureXR.GetBlackTexture());
```

Comment:
Passing black here, won't make the check in here https://github.com/Unity-Technologies/Graphics/pull/5536/files#diff-897ad540cb7032a41aaf5e63dce2b5256798820e918184f5ac4db09ac32c70f9R847 always early out? Is that ok?

Reply:
You are right, doesn't work (the tests also failed). Not sure how to pas a 1e10 texture tho haha.